### DISCUSSION TITLE
Donations

### DISCUSSION THREAD

**Discussion by Aragas** [elringus/bootsharp#11] [General]
There is an issue with PayPal that restricts Russia<-->Russia transactions from being made!

We wanted to do small monthly donations for DotNetJS, as a gesture of gratitute for maintaining a general WASM workflow, not browser-only like MS does, but because mostly I manage the donations and I do them from my Russian PayPal account, it's not possible to setup the donations!

Do you know if there is any workaround for this? Maybe an alternative system for donations?

**Comment by elringus** [comment]
Hey, Thanks for using DotNetJS and the will to support the project! I’m not aware of a way to overcome the PayPal limitation, unfortunately. That said, I’ll probably be able to replace ko-fi/PayPal with GitHub Sponsors soon, which shouldn’t have such limitations.
